helpers/installcredprovider.ps1

```diff
@@ -111,13 +113,20 @@ function InstallZip {
         $releaseJson = $release.Content | ConvertFrom-Json
         $zipAsset = $releaseJson.assets | ? { $_.name -eq $zipFile }
         $packageSourceUrl = $zipAsset.browser_download_url
-    } catch {
-        Write-Error $zipErrorString
-        return
+                
+        if (!$releaseJson) {
+            throw("Unable to convert JSON")
+        }
+        if (!$zipAsset) {
+            throw("Unable to retrieve zip asset")
```

Comment:
nit: retrieve sounds like an API call to me, we're just reading the json here so maybe better would be something like "Unable to find asset $zipFile from release json object", and for the packageSourceUrl something like, "Unable to find download url from asset $zipFile"